Furball.Vixie.TestApplication/Program.cs

```diff
@@ -8,8 +9,10 @@
 
 options.VSync = false;
 
-GraphicsBackend.PrefferedBackends = Backend.OpenGL41 | Backend.OpenGL20;
+GraphicsBackend.PrefferedBackends = Backend.Direct3D11;
 
 VeldridBackend.PrefferedBackend = Veldrid.GraphicsBackend.Vulkan;
 
-new TestGame(options).Run(options);
\ No newline at end of file
+//Thread.Sleep(7500);
+
+new TestGame(options).Run(options, GraphicsBackend.PrefferedBackends);
```

Comment:
This extra param and that comment should be removed

Reply:
now that it actually works, it can yeah